net/proftpd/Portfile

```diff
@@ -32,17 +35,20 @@ master_sites        ftp://ftp.proftpd.org/distrib/source/ \
                     ftp://ftp.solnet.ch/mirror/proftpd.org/distrib/source/ \
                     gentoo
 
-depends_build       port:libtool
+# remove autoconf when updating to 1.3.10
+depends_build       port:libtool \
+                    port:autoconf
```

Comment:
At least `autoconf` dependency should be already added by `use_autoconf`. What does `port lint` say?

Reply:
```
#=> port lint proftpd
--->  Verifying Portfile for proftpd
--->  0 errors and 0 warnings found.
```

Reply:
@reneeotten Doesn’t `use_autoconf` imply a dependency automatically?

Reply:
it does imply the dependency on port:autoconf - removing it
```
#=> port -cuN uninstall proftpd
#=> port -cuN uninstall autoconf
#=> port install proftpd
--->  Computing dependencies for proftpd
The following dependencies will be installed:  autoconf
Continue? [Y/n]: 
--->  Fetching archive for autoconf
```